### PR TITLE
Added info on the HF supported Linux packages.

### PR DESCRIPTION
**Pull request by andyli**
Please proofread ;)

### CODE REVIEW

File: www/view/download/linux.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<div class="section">	
+	<div class="container">
+		<h1>Linux Software Packages</h1>
+		The Haxe Foundation officially participates in the maintenance of Haxe and Neko packages for a number of popular Linux distributions. It is recommended to use those packages whenever available.
```
Comment (Simn) [inline www/view/download/linux.html]: "whenever" sounds strange here, "if" would be better

File: www/view/download/linux.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		<p>We recommend you to use our Personal Package Archive (PPA), <a href="https://launchpad.net/~haxe/+archive/ubuntu/releases" target="_blank">ppa:haxe/releases</a>, which provides the latest versions of Haxe and Neko for all the currently supported Ubuntu versions. To install Haxe and Neko using the PPA, run the commands as follows:</p>
+		<pre>sudo add-apt-repository ppa:haxe/releases -y
+sudo apt-get update
+sudo apt-get install haxe -y
+mkdir ~/haxelib &amp;&amp; haxelib setup ~/haxelib</pre>
+		<p>Neko will be installed as a dependency of Haxe.</p>
+		<p>Note that the PPA is also usable for Ubuntu based distributions, e.g. Linux Mint.</p>
+
+		<h2>Debian</h2>
+		<p>The Haxe Foundation continuously updates the Haxe and Neko packages in the Debian archive. However, due to the long release cycle of Debian. It is very likely that only the packages in the testing channel and the unstable (sid) channel are up-to-date.</p>
```
Comment (Simn) [inline www/view/download/linux.html]: The second sentence is incomplete, you probably meant to have a comma after "Debian".

File: www/view/download/linux.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+		<p>To install the currently available versions for your system, simply run the commands as follows:</p>
+		<pre>sudo dnf install haxe -y
+mkdir ~/haxelib &amp;&amp; haxelib setup ~/haxelib</pre>
+		<p>Neko will be installed as a dependency of Haxe.</p>
+
+		<h2>openSUSE</h2>
+		<p>The Haxe Foundation has a devel project, named <a href="https://build.opensuse.org/project/show/devel:languages:haxe" target="_blank">devel:languages:haxe</a>, in the openSUSE Build Service. The project provides the latest versions of Haxe and Neko for all the currently supported openSUSE versions.</p>
+		<p>To install Haxe and Neko from the devel project, visit the <a href="https://build.opensuse.org/package/show/devel:languages:haxe/haxe" target="_blank">Haxe package page</a>, click "Download package" at the top-right corner and follow the instruction there. Neko will be installed as a dependency of Haxe.</p>
```
Comment (Simn) [inline www/view/download/linux.html]: Should probably be "instructions".

**Comment by andyli** [comment]
Have just fixed the grammar issues. Thanks @Simn!

**Pull request merged**